styles/menu.css

```diff
@@ -0,0 +1,127 @@
+.quiz-main-menu {
+  display: flex;
+  position: fixed;
+  width: 225px;
+  height: 100vh;
+  top: 0;
+  left: -225px;
+  box-sizing: border-box;
+  background: rgba(0, 0, 0);
+  border-right: solid var(--starwars-yellow) 0.15rem;
+  flex-direction: column;
+  transition: transform 0.3s;
+}
+
+.shadow {
+  box-shadow: 1px 1px 120px var(--starwars-yellow);
+}
+
+.btn {
+  font-size: 28px;
+  background: rgba(255, 255, 255, 0.05);
+  border: 2px solid rgba(0, 0, 0, 0.4);
+  border-top: white solid 2px;
+  border-bottom: white solid 2px;
+  text-align: left;
+  color: white;
+  margin: 0.2em 0;
+  padding: 0.5em;
+}
+
+.btn:hover {
+  font-size: 28px;
+  color: var(--starwars-yellow);
+  box-shadow: 1px 1px 100px var(--starwars-yellow);
+}
+
+.btn:active {
+  background-color: rgba(255, 255, 255, 0.25);
+  color: white;
+  border-color: rgba(255, 255, 255, 0.25);
+  box-shadow: 1px 1px 100px white;
+}
+
+.burger {
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 10px 20px;
+  font-size: 25px;
+  background: none;
+  border: none;
+  cursor: pointer;
+  color: #fff;
+  transition: transform 0.3s;
+}
+
+.selectMode {
+  color: #fff;
+  font-size: 25px;
+  padding: 0.5em;
+  font-weight: bold;
+  font-family: Montserrat, sans-serif;
+}
+
+@media screen and (min-width: 360px) {
+  #main-grid-container {
+    display: block;
+  }
+
+  .sw-logo {
+    margin: 2em auto 3em auto;
+  }
+
+  .btn {
+    font-size: 20px;
+  }
+
+  .btn:hover {
+    font-size: 20px;
+  }
+  .active {
+    transform: translateX(225px);
+    transition: transform 0.3s;
+  }
+  .hide {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .burger {
+    font-size: 26px;
+  }
+
+  .quiz-main-menu {
+    width: 260px;
+    left: -260px;
+    border-right: solid var(--starwars-yellow) 0.15rem;
+  }
+
+  .btn {
+    font-size: 22px;
+  }
+
+  .active {
+    transform: translateX(260px);
+  }
+  .btn:hover {
+    font-size: 22px;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .burger {
+    font-size: 36px;
+  }
+
+  .quiz-main-menu {
+    width: 350px;
+    left: -350px;
+    border-right: solid var(--starwars-yellow) 0.2rem;
+  }
+
+  .active {
+    transform: translateX(350px);
+  }
+}
```

Comment:
I see that you removed styles for wider screens and it causes the mode options to be accessed only by opening the side menu. I think the mode options should be available right away and the side menu should not be accessible on wider screens.